src/explore-education-statistics-frontend/next-sitemap.config.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
         ],
       },
     ],
-    additionalSitemaps: [`${process.env.PUBLIC_URL}server-sitemap.xml`],
+    additionalSitemaps: [`${process.env.PROD_PUBLIC_URL}server-sitemap.xml`],
```

Comment:
Don't we need to make additional changes in the `server-sitemap.xml` as well? All the locations are using `PUBLIC_URL` instead of `PROD_PUBLIC_URL`

Reply:
Hah, yeah happily. I went with the slash to be consistent with `PUBLIC_URL` but yes would much prefer without